test/check-application

```diff
@@ -920,15 +923,17 @@ class TestFiles(testlib.MachineCase):
 
         # Copy/paste directory
         self.browser.wait_not_present(".pf-c-empty-state")
-        m.execute("runuser -u admin mkdir /home/admin/copyDir")
-        b.click("[data-item='newfile']")
+        m.execute("runuser -u admin mkdir /home/admin/copyDir; chown admin:users /home/admin/copyDir")
+        b.click("[data-item='copyDir']")
         b.click("#dropdown-menu")
         b.click("#copy-item")
         b.mouse("[data-item='newdir']", "dblclick")
         b.click("#dropdown-menu")
         b.click("#paste-item")
-        b.wait_visible("[data-item='newfile']")
-        self.assertEqual(m.execute("head -n 1 /home/admin/newdir/newfile"), "test_text\n")
+        b.wait_visible("[data-item='copyDir']")
+        b.click("[data-item='copyDir']")
+        b.wait_text("#description-list-owner dd", "admin")
+        b.wait_text("#description-list-group dd", "users")
```

Comment:
I don't understand this. The hunk above changes the *copy* source from 'newfile' to 'copyDir' -- but here you again loko at the copy source, not the pasted version in newdir? (This is a bit hard to read and could use some comment)